tendermint/src/rpc/endpoint/subscribe.rs

```diff
@@ -29,6 +29,9 @@ impl rpc::Request for Request {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Response {}
 
+/// Subcribe is weird RPC endpoint. It's only meaningful at websocket response and there isn't a
+/// synchronous reponse offered. It there is an error it's asynchronous and we don't try and stich
+/// the async response backer together with the request.
```

Comment:
```suggestion
/// the async response back together with the request.
```
So that's why you can also ignore the error here: https://github.com/informalsystems/tendermint-rs/pull/225#discussion_r429497738
Interesting. The documentation isn't as clear as it should be https://docs.tendermint.com/master/rpc/#/Websocket/subscribe

Thanks for clarifying this!